file/test/sentry_io_overrides_integration_test.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:io';
+
+import 'package:sentry/sentry.dart';
+import 'package:sentry_file/sentry_file.dart';
+import 'package:sentry_file/src/sentry_io_overrides.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+import 'mock_sentry_client.dart';
+
+void main() {
```

Comment:
Add tests that actually when `createFile` now is called, it returns a `SentryFile`.